shell/platform/darwin/ios/framework/Source/FlutterSemanticsScrollView.mm

```diff
@@ -0,0 +1,138 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterSemanticsScrollView.h"
+
+#import "flutter/shell/platform/darwin/ios/framework/Source/SemanticsObject.h"
+
+@interface FlutterSemanticsScrollView ()
+@property(nonatomic, assign) SemanticsObject* semanticsObject;
+@end
+
+@implementation FlutterSemanticsScrollView
+
+- (instancetype)initWithSemanticsObject:(SemanticsObject*)semanticsObject {
+  self = [super initWithFrame:CGRectZero];
+  if (self) {
+    _semanticsObject = semanticsObject;
```

Comment:
I believe we're missing a retain here, which is the cause of a crash noted in https://github.com/flutter/flutter/issues/92853

Reply:
It was not that we're missing a retain, but that we're missing niling this out when the parent object is deallocated.